src/range_map.rs

```diff
@@ -63,7 +63,7 @@ impl<T> RangeMap<T> {
     /// through interior mutability.
     ///
     /// The iterator also provides the offset of the given element.
```

Comment:
```suggestion
    /// The iterator also provides the range of the given element.
    /// How exactly the ranges are split can differ even for otherwise identical
    /// maps, so user-visible behavior should never depend on the exact range.
```

Reply:
(same for `iter_mut`)

Reply:
Marking as unresolved since the extended comments I asked for are not present in the current PR.